boards/pico_explorer_base/Makefile

```diff
@@ -22,14 +25,16 @@ flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
 	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<;  reset; shutdown;"
```

Comment:
Probably doesn't make sense for flash-debug to still use openocd when flash no longer does

Reply:
I would not delete this, as someone might want to use it. What about renaming it to `flash-openocd`?

Reply:
Sure, in that case I would modify it to use the release binary though

Reply:
Done.